File: AOJ/ITP1/ITP1_4_A_A_B_Problem.py
a,b=map(int,input().split());print(a//b,a%b,f'{a/b:f}')
"""
a, b = map(int, input().split())
d, r = divmod(a, b)
f = a / b
print(f"{d} {r} {f:.5f}")
"""

File: AOJ/ITP1/csharp/ITP1_4_A_A_B_Problem.cs
using System;
using System.Linq;
using static System.Console;

class P
{
    static void Main()
    {
        var arr = ReadLine().Split().Select(int.Parse).ToList();
        int a = arr[0], b = arr[1];
        WriteLine($"{a / b} {a % b} {(double) a / b :f5}");
    }
}
